apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createservice ticket.ResidentTicketService --type mutations`
+ */
+const { makeClientWithProperty } = require('../../property/utils/testSchema')
+const { createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
+const { makeLoggedInAdminClient, makeClient } = require('@core/keystone/test.utils')
+const { createResidentTicketByTestClient } = require('@condo/domains/ticket/utils/testSchema')
+const { UUID_RE } = require('@core/keystone/test.utils')
+
+describe('CreateResidentTicketService', () => {
+    test('user: create resident ticket', async () => {
+        const client = await makeClientWithProperty()
```

Comment:
We should create resident client :)

Reply:
But such tickets can be created by users who are not yet residents, right?